cypress/e2e/dashboard/events/events-index.cy.ts

```diff
@@ -316,4 +316,26 @@ describe('events dashboard', () => {
     cy.reload();
     cy.updateEvent(eventId, eventData).then(expectToBeRejected);
   });
+
+  it('chapter admin should be allowed to delete event, but nobody else', () => {
+    const eventId = 1;
+    // admin of chapter 1
+    cy.login(Cypress.env('JWT_ADMIN_USER'));
+    cy.reload();
+    cy.deleteEvent(eventId).then((response) => {
+      expect(response.body.errors).not.to.exist;
+    });
```

Comment:
Might be better to move a allowed deletion try at the end of tests. It's a bit strange to check if deletion request is rejected by authorization, on event that doesn't exists anymore.